app/src/marketplace/components/StreamPreviewPage/index.jsx

```diff
@@ -118,11 +120,18 @@ class StreamPreviewPage extends React.Component<Props, State> {
         })
     }
 
+    hasData = () => {
+        this.setState(() => ({
+            hasData: true,
+        }))
+    }
```

Comment:
state key and function name being the same took me a moment to figure out if it was a mistake, suggest changing one of them

Reply:
good point, will follow up with a cleanup PR